packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js

```diff
@@ -55,6 +55,7 @@ const GreyH6 = styled(({ theme }) => ({
 const StyledCardContent = styled(({ theme }) => ({
   alignItems: 'center',
   marginBottom: theme.sizing.baseUnit,
+  paddingVertical: 0,
```

Comment:
Why are we adding padding here?

Reply:
@redreceipt Why do you hate padding? ;)

Just kidding. I removed it. I have no idea why that was there.